### ISSUE TITLE
failed to put blob since update

### ISSUE THREAD

**Issue by ramonskie** [pivotal-cf/azure-blobstore-resource#13]
since the changes from February an upload to the blobstore fails.

we have 
```
- name: test-image-path
  type: azure-blob
  source:
    container: pipeline
    storage_account_name: ((azure-storage-account-name))
    storage_account_key:   ((azure-storage-access-key))
    versioned_file: test-image-path

- task foobar
  - put: test-image-path
    params:
      file: packer_artifacts/vhd_uri
```

as i see in the azure blobstore it created a blob with the name `vhd_uri`
but should update the file `test-image-path`

concourse replies with
```
2019/02/08 10:34:28 failed to copy blob: storage: service returned error: StatusCode=404, ErrorCode=404 The specified blob does not exist., ErrorMessage=no response body was available for error status code, RequestInitiated=Fri, 08 Feb 2019 10:34:28 GMT, RequestId=32ea0716-801e-007d-2799-bfb435000000, API Version=2016-05-31, QueryParameterName=, QueryParameterValue=
```

**Comment by ramonskie** [comment]
it seems that if the file name is different than the versioned_file it fails.

**Comment by christianang** [comment]
Good find. Fixed in 674d7c1da4148220a535c456fd006030cb78aae5. It should now use the `versioned_file` param for the blob name and not the filename. Reopen if I missed something and it is still not working.

**Comment by ramonskie** [comment]
thanks for the quick fix
